frontend/package.json

```
@@ -10,11 +10,23 @@
     "lint": "eslint ."
   },
   "dependencies": {
+    "@radix-ui/react-accordion": "^1.2.11",
+    "@radix-ui/react-checkbox": "^1.3.2",
+    "@radix-ui/react-dialog": "^1.1.14",
+    "@radix-ui/react-dropdown-menu": "^2.1.15",
+    "@radix-ui/react-label": "^2.1.7",
+    "@radix-ui/react-scroll-area": "^1.2.9",
+    "@radix-ui/react-select": "^2.2.5",
+    "@radix-ui/react-separator": "^1.1.7",
     "@radix-ui/react-slot": "1.2.2",
+    "@radix-ui/react-switch": "^1.2.5",
+    "@radix-ui/react-tabs": "^1.1.12",
+    "@radix-ui/react-toggle": "^1.1.9",
     "@react-router/node": "7.5.3",
     "@react-router/serve": "7.5.3",
     "@sentry/browser": "9.19.0",
     "@sentry/react": "9.19.0",
+    "@tanstack/react-table": "^8.21.3",
```

```suggestion
    "@radix-ui/react-accordion": "1.2.11",
    "@radix-ui/react-checkbox": "1.3.2",
    "@radix-ui/react-dialog": "1.1.14",
    "@radix-ui/react-dropdown-menu": "2.1.15",
    "@radix-ui/react-label": "2.1.7",
    "@radix-ui/react-scroll-area": "1.2.9",
    "@radix-ui/react-select": "2.2.5",
    "@radix-ui/react-separator": "^1.1.7",
    "@radix-ui/react-slot": "1.2.2",
    "@radix-ui/react-switch": "1.2.5",
    "@radix-ui/react-tabs": "1.1.12",
    "@radix-ui/react-toggle": "1.1.9",
    "@react-router/node": "7.5.3",
    "@react-router/serve": "7.5.3",
    "@sentry/browser": "9.19.0",
    "@sentry/react": "9.19.0",
    "@tanstack/react-table": "8.21.3",
```
Please pin the version

---

Not sure what you mean.